arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -745,9 +745,9 @@ static void mbm_update(struct rdt_resource *r, struct rdt_domain *d,
 void cqm_handle_limbo(struct work_struct *work)
 {
 	unsigned long delay = msecs_to_jiffies(CQM_LIMBOCHECK_INTERVAL);
-	int cpu = smp_processor_id();
 	struct rdt_resource *r;
 	struct rdt_domain *d;
+	int cpu;
```

Comment:
upstream they do not have cpu they use d->cqm_work_cpu (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=a4846aaf39455fe69fce3522b385319383666eef)

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev